include/mbedtls/config.h

```diff
@@ -3863,6 +3863,25 @@
 
 /* \} SECTION: Compile-time SSL configuration */
 
+/**
+ * \def MBEDTLS_AES_ONLY_128_BIT_KEY_LENGTH
+ *
+ * Use only 128-bit keys in AES operations.
+ *
+ * Uncommenting this macro removes support for AES operations that are using 192
+ * or 256-bit keys.
+ *
+ * Tradeoff: Uncommenting this macro reduces ROM footprint by ~200 bytes.
+ *
+ * If uncommented, uncomment also MBEDTLS_CTR_DRBG_USE_128_BIT_KEY
```

Comment:
I would add this to `config.h`

Reply:
I meant `check_config.h` . Sorry...

Reply:
Np, I was planning to ask about that once the CI tests are all green. 
The check is now added to `check_config.h`.